src/core/highlight/highlight.view.js

```diff
@@ -117,67 +118,69 @@ export default class HighlightView extends View {
 	}
 
 	highlight(key, cls) {
+		const table = this.table;
+		const markup = this.markup;
 		const index = this.columnIndex(key);
 		if (index < 0) {
 			return noop;
 		}
 
-		const head = this.markup.head;
-		if (head.rows.length) {
-			for (let row of head.rows) {
-				row.cells[index].classList.add(`${GRID_PREFIX}-${cls}`);
-				if (index > 0) {
-					row.cells[index - 1].classList.add(`${GRID_PREFIX}-${cls}-prev`);
-				}
+		const head = table.head;
+		if (markup.head.rows.length) {
+			const cells = head.column(index).cells();
+			cells.forEach((cell) => cell.addClass(`${GRID_PREFIX}-${cls}`));
+			if (index > 0) {
+				const cells = head.column(index - 1).cells();
+				cells.forEach((cell) => cell.addClass(`${GRID_PREFIX}-${cls}-prev`));
+			}
 
-				if (index < head.rows.length - 1) {
-					row.cells[index + 1].classList.add(`${GRID_PREFIX}-${cls}-next`);
-				}
+			if (index < markup.head.rows.length - 1) {
+				const cells = head.column(index + 1).cells();
+				cells.forEach((cell) => cell.addClass(`${GRID_PREFIX}-${cls}-next`));
 			}
 		}
-
-		const body = this.markup.body;
-		for (let row of body.rows) {
-			row.cells[index].classList.add(`${GRID_PREFIX}-${cls}`);
+		if (markup.body.rows.length) {
+			const cells = table.body.column(index).cells();
+			cells.forEach((cell) => cell.addClass(`${GRID_PREFIX}-${cls}`));
 		}
-
-		const foot = this.markup.foot;
-		for (let row of foot.rows) {
-			row.cells[index].classList.add(`${GRID_PREFIX}-${cls}`);
+		if (markup.foot.rows.length) {
+			const cells = table.foot.column(index).cells();
+			cells.forEach((cell) => cell.addClass(`${GRID_PREFIX}-${cls}`));
 		}
 
 		return this.blur(key, cls);
 	}
 
 	blur(key, cls) {
+		const table = this.table;
+		const markup = this.markup;
 		const index = this.columnIndex(key);
 		if (index < 0) {
 			return noop;
 		}
 
 		return () => {
-			const head = this.markup.head;
-			if (head.rows.length) {
-				for (let row of head.rows) {
-					row.cells[index].classList.remove(`${GRID_PREFIX}-${cls}`);
-					if (index > 0) {
-						row.cells[index - 1].classList.remove(`${GRID_PREFIX}-${cls}-prev`);
-					}
-
-					if (index < head.rows.length - 1) {
-						row.cells[index + 1].classList.remove(`${GRID_PREFIX}-${cls}-next`);
-					}
+			const head = table.head;
+			if (markup.head.rows.length) {
```

Comment:
Unfortunately, I face the TypeError: Cannot read property 'cells' of undefined if we haven't this condition on th hover